### ISSUE TITLE
Finalize TimeControl PGN header support

### ISSUE THREAD

**Issue by niklasf** [niklasf/python-chess#1033]
After #1032:

* [ ] Implement setter

**Comment by youssouf1994** [comment]
I am trying to understand the task on this issue and may be implementing it. But I don't understand where do you want to add a setter? Is it in the Game class or other?

**Comment by niklasf** [comment]
Yes, next to https://github.com/niklasf/python-chess/blob/e773cd4eb93519397fe3eea0a8f92cc4662c718c/chess/pgn.py#L896
